src/resources/postcss/components/_search.pcss

```diff
@@ -48,18 +48,18 @@
 	.tribe-common-c-search {
 		@media (--viewport-medium) {
 			display: flex;
+			width: 100%;
```

Comment:
My main question around this is that `.tribe-common-c-search` should be a standalone component, and all styles should be contained within the `.tribe-common-c-search` component. How this component is used and fit into the flow of the document shouldn't be defined in the component. Rather, we can define this in `.tribe-events-c-events-bar__search` (or another class, but the `<form>` element) to define how the search component fits into the flow of the events bar. Not likely, but if we end up using this component somewhere else, the width and how this component flows within that markup may be different.

Reply:
Ok, so you would set the width directly on the first child of `.tribe-events-c-events-bar__form` and not this class directly.

Reply:
Yes, whatever the `<form>` element class is in `.tribe-events-c-events-bar`. I don't know what the class is off hand, but if there isn't one, define one next to `.tribe-common-c-search`.